src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  width: 16px;
+  margin-right: 4px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-image: url(./images/star.svg);
+  background-position: center;
+}
+
+.star:last-child {
+  margin-right: 0;
+}
+
+.star--0 :nth-child(-n + 0),
+.stars--1 :nth-child(-n + 1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
```

Comment:
There is a typo in the class selector `.star--0 :nth-child(-n + 0)`. The space between `.star--0` and `:nth-child` is incorrect. It should be `.stars--0:nth-child(-n + 0)` to correctly target the child elements within the `.stars--0` container. This issue is present for all similar selectors in this block.